### PR TITLE
API Documentation

### PR DESCRIPTION
**Pull request by akhuoa**
1. API documentation for Gallery component.
2. GitHub Actions workflow for API documentation.
3. Update dev server port for local development.

### CODE REVIEW

**Comment by ddjnw1yu** [comment]
Looks good to me. Thanks. You probably need to ask @alan-wu to merge your PR.

**Review by ddjnw1yu — APPROVED** [review]
Looks good to me. Thanks. You probably need to ask @alan-wu to merge your PR.

File: src/components/Gallery.vue [hunk 1]
```diff
@@ -16,56 +16,87 @@ function convertRemToPixels(rem) {
...
+  /**
+   * The CSS for image style in card.
+   */
   imageStyle: {
     type: Object,
     default: () => {
       return {}
     },
   },
+  // TODO: not in use
```
Comment (Tehsurfer) [inline src/components/Gallery.vue:99]: I think this prop is causing some strange results in the documentation generation:

image

**Review by Tehsurfer — CHANGES_REQUESTED** [review]
I would remove the unused props from the documentation or add a not "Unused" to them.

Other than that, the other fields look right to me and the docs generate and run well :)

File: src/components/Gallery.vue (hunk 1 above)
Comment (akhuoa) [reply src/components/Gallery.vue:99]: It is fixed in this commit https://github.com/ABI-Software/mapcore-gallery/pull/9/commits/824fbc24fda8a596337b60b17bc96ca5df0b4728

**Comment by Tehsurfer** [comment]
Apparently it's just my laptop it doesn't work on but if it works on github actions than that doesn't really matter. We will set up the action and merge

**Pull request merged**